x/bridge/types/assets_test.go

```diff
@@ -0,0 +1,83 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+// TestAsset tests if Asset is properly validated.
+func TestAsset(t *testing.T) {
+	testCases := []struct {
+		name          string
+		asset         types.Asset
+		expectedValid bool
```

Comment:
For future reference, we like to match errors instead of boolean expected errors. This way we can be certain we are failing for the expected reason, and it promotes the use of defined errors.

Reply:
Done for all tests